pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -779,6 +779,9 @@ type GaneshaServerSpec struct {
 
 	// PriorityClassName sets the priority class on the pods
 	PriorityClassName string `json:"priorityClassName,omitempty"`
+
+	// LogLevel set logging level
+	LogLevel string `json:"logLevel,omitempty"`
```

Comment:
If there is only debug or non-debug, let's make it a bool
```suggestion
	EnableDebugLogging string `json:"enableDebugLogging,omitempty"`
```

Reply:
according to the comments we want all the log level so, keeping it a string.